brave/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright 2016-2020 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.zipkin.reporter2</groupId>
+    <artifactId>zipkin-reporter-parent</artifactId>
+    <version>2.12.4-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>zipkin-reporter-brave</artifactId>
+  <name>Zipkin Reporter Spring Factory Brave</name>
+  <description>Allows you to configure Zipkin Reporter using XML</description>
+
+  <properties>
+    <!-- Matches Export-Package in bnd.bnd -->
+    <module.name>zipkin2.reporter.brave</module.name>
+
+    <main.basedir>${project.basedir}/..</main.basedir>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>zipkin-reporter</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.zipkin.brave</groupId>
+      <artifactId>brave</artifactId>
+      <!-- Don't pin Brave -->
```

Comment:
Don't we need to at least define some min Brave version though?

Reply:
not worth it, imho and we don't do the same anywhere else either, not in brave JMS for example, or any other project.

if we do compile scope vs provided here, it is more likely to end up as a classpath clash in our gitter channel and I really don't want to do that.

Reply:
ps the caution in general is about Nteen build tools and some dimensions of all of them and how they interpret Maven variable version range. I personally can't take on that risk especially knowing we knocked all things like this provided to dig out of problems in the first place. Not something I want to waranty, and likely it would be me.

this treats brave version conventionally like literally everything else saying the primary library owns that version. The only difference is I put a comment to that nature, where in other poms it is implied.